sky/utils/command_runner.py

```diff
@@ -42,13 +42,16 @@ def _ssh_control_path(ssh_control_filename: Optional[str]) -> Optional[str]:
     return path
 
 
-def ssh_options_list(ssh_private_key: Optional[str],
-                     ssh_control_name: Optional[str],
-                     *,
-                     ssh_proxy_command: Optional[str] = None,
-                     docker_ssh_proxy_command: Optional[str] = None,
-                     timeout: int = 30,
-                     port: int = 22) -> List[str]:
+def ssh_options_list(
+    ssh_private_key: Optional[str],
+    ssh_control_name: Optional[str],
+    proxy_to_k8s: Optional[bool] = False,
```

Comment:
It seems to me the only purpose of this flag `proxy_to_k8s` is to disable ControlMaster. Can we rename this to `disable_control_master` so it's easier to parse and reusable in the future?